geomesa-quickstart-nifi/README.md

```diff
@@ -0,0 +1,215 @@
+example then you may skip this section. Simply copy the data you have into
+/tmp/gdelt.
+
+The [GDELT Event database](http://www.gdeltproject.org) provides a
+comprehensive time- and location-indexed archive of events reported in
+broadcast, print, and web news media worldwide from 1979 to today. You
+can download raw GDELT data files at
+http://data.gdeltproject.org/events/index.html. For example, the
+following Bash commands use ``wget`` to download all event files for
+2013 (2 GB compressed, 14 GB uncompressed) and ``md5sum`` to
+verify each by their MD5 hash:
+
+```bash
+$ mkdir /tmp/gdelt && cd /tmp/gdelt 
+$ wget http://data.gdeltproject.org/events/md5sums 
+$ for file in `cat md5sums | cut -d' ' -f3 | grep '^2013'` ; do wget http://data.gdeltproject.org/events/$file ; done 
+$ md5sum -c md5sums 2>&1 | grep '^2013' 
+$ unzip '*.zip'
+```
+
+Download and Build the GeoMesa NiFi project
+-------------------------------------------
+
+Pick a reasonable directory on your machine, and run:
+
+```bash
+$ git clone https://github.com/geomesa/geomesa-nifi.git
+$ cd geomesa-nifi
+```
+
+To build, run
+
+```bash
+$ mvn clean install
+```
+
+This will build three processors. 
+
+* PutGeoMesa - Ingest data into GeoMesa with a GeoMesa converter or from geoavro
+* PutGeoTools - Ingest data into an arbitrary GeoTools Datastore based on parameters using a GeoMesa converter or avro
+* ConvertToGeoAvro - Use a GeoMesa converter to create geoavro
+
+However we will only be using the PutGeoMesa for this QuickStart.
+
+Install the GeoMesa Processor
+-----------------------------
+
+### Install the GeoMesa NiFi Processor
+
+In order to use NiFi with GeoMesa we need to first install the GeoMesa 
+processor. To do this simply copy the ``geomesa-nifi-nar-$VERSION.nar``
+that you just built from ``geomesa-nifi/geomesa-nifi-nar/target`` to the
+``lib/`` directory of you NiFi installation.
+  
+### Install the SFTs and Converters
+
+Additionally we need to copy the ``geomesa-accumulo-$VERSION-data.jar``
+from the geomesa-accumulo-dist to the ``lib/`` of the NiFi installation. 
+The ``geomesa-accumulo-$VERSION-data.jar`` is located in the ``dist/sfts-converters``
+directory of the geomesa-accumulo-dist. This jar contains the sfts and 
+converters needed for GeoMesa to ingest the GDELT data. You can obtain 
+the geomesa-accumulo-dist from [here](https://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-accumulo-dist_2.11/1.3.1/geomesa-accumulo-dist_2.11-1.3.1-bin.tar.gz) 
+or you may build it locally if you have geomesa source available using: 
+
+```bash
+$ mvn clean install -am -pl :geomesa-accumulo-dist_2.11
+```
+
+Upload the NiFi flow template
+-----------------------------
+
+The NiFi template provided [here](../assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml)
+sets up a basic NiFi flow from a GetFile processor to a PutGeoMesa processor. 
```

Comment:
FYI, it's a template that you upload with an import template option. Then it's populated in a templates _thing_ that you can import like a processor.